src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,195 @@
+import csv
+import getopt
+import json
+import os
+import os.path
+import serialobj
+import sys
+sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))
```

Comment:
That script would be xunitrunner-perf.py which needs some changes before I create a PR for.